File: create_dirs.py
import os
base = r"e:\Nam_3_HK2\ThiGiac\Fitness\src"
os.makedirs(os.path.join(base, "components"), exist_ok=True)
os.makedirs(os.path.join(base, "utils"), exist_ok=True)
os.makedirs(os.path.join(base, "hooks"), exist_ok=True)
print("Directories created including hooks")

File: create-dirs.js
const fs = require('fs');
const path = require('path');

const baseDir = __dirname;
const dirs = [
  path.join(baseDir, 'src'),
  path.join(baseDir, 'src', 'components'),
  path.join(baseDir, 'src', 'utils')
];

dirs.forEach(dir => {
  if (!fs.existsSync(dir)) {
    fs.mkdirSync(dir, { recursive: true });
    console.log(`Created: ${dir}`);
  } else {
    console.log(`Already exists: ${dir}`);
  }
});

console.log('Directory creation complete!');
